google-cloud-clients/google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta1/it/ITBigQueryStorageTest.java

```diff
@@ -104,6 +151,57 @@ public void testSimpleRead() {
     assertEquals(164_656, avroRowCount);
   }
 
+  @Test
+  public void testSimpleReadAndResume() {
```

Comment:
What is the end user use case for resume? Shouldn't this be handled transparently by the client?

Reply:
@kmjung , it is handled transparently by the client. The goal of this test is to explicitly verify it.

Reply:
Resuming by offset can also be used explicitly by clients in scenarios:

1. The row progress per stream is stored in customers systems
1. Customer pauses the stream reading for applying changes on their system.
1. Customer resumes processing.

I thought it would be good to have an explicit tests for this functionality.